### PR TITLE
 Ceph s3 tests updates

### PR DESCRIPTION
**Pull request by romayalon**
### Explain the changes
1. fixed regression tests that can be fixed.
2. updated blacklist.
3. updated ceph tests config file.
4. fixed test_ceph_s3.js file to cope with tenant.

### Issues: Fixed #xxx / Gap #xxx
1. 

### Testing Instructions:
1.

### CODE REVIEW

**Comment by nimrod-becker** [comment]
@liranmauda @romayalon 
Might be good to sync on the eslint-ignore since it fails lint here as well

File: src/test/system_tests/ceph_s3_config.conf [hunk 1]
```diff
@@ -6,7 +6,7 @@
 host = localhost
 
 ## uncomment the port to use something other than 80
```
Comment (liranmauda) [inline src/test/system_tests/ceph_s3_config.conf]: we should change the comment or remove it

**Review by liranmauda — APPROVED** [review]

File: src/endpoint/s3/s3_rest.js [hunk 2]
```diff
@@ -315,7 +315,7 @@ function get_bucket_and_key(req) {
     return {
         bucket,
         // decode and replace hadoop _$folder$ in key
-        key: decodeURIComponent(key).replace(/_\$folder\$/, '/'),
+        key: decodeURIComponent(key.replace('%', '%25')).replace(/_\$folder\$/, '/'),
```
Comment (jeniawhite) [inline src/endpoint/s3/s3_rest.js]: Not sure about that fix. It looks like something that will hunt us in the future.
It usually not a good practice to just replace characters in a request.

File: src/endpoint/s3/ops/s3_put_object.js [hunk 3]
```diff
@@ -37,7 +37,7 @@ async function put_object(req, res) {
         md_conditions: http_utils.get_md_conditions(req),
         source_md_conditions: http_utils.get_md_conditions(req, 'x-amz-copy-source-'),
         xattr: s3_utils.get_request_xattr(req),
-        xattr_copy: (req.headers['x-amz-metadata-directive'] === 'COPY'),
+        xattr_copy: (req.headers['x-amz-metadata-directive'] !== 'REPLACE'),
```
Comment (jeniawhite) [inline src/endpoint/s3/ops/s3_put_object.js]: What is the difference? There are only two variables COPY and REPLACE. Am I'm missing something?

Comment (nimrod-becker) [reply src/endpoint/s3/ops/s3_put_object.js]: The case where it's not being sent, i.e. default behavior

File: src/endpoint/s3/s3_rest.js (hunk 2 above)
Comment (romayalon) [reply src/endpoint/s3/s3_rest.js]: fixed, see #5814 pull request.

**Pull request merged**